ui/App.js

```diff
@@ -7,7 +7,8 @@ const options = [
   { value: 'ropsten', label: 'Ropsten' },
   { value: 'rinkeby', label: 'Rinkeby' },
   { value: 'kovan', label: 'Kovan' },
-  { value: 'goerli', label: 'Görli' }
+  { value: 'goerli', label: 'Görli' },
+  { value: 'localhost', label: 'localhost:8545' }
```

Comment:
This is here just for convenience while developing. At some point will need to figure out how to add it conditionally though because it can't be a real option.